metadata.rb

```diff
@@ -4,7 +4,7 @@
 license          'Apache 2.0'
 description      'Installs/Configures pdns'
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version          '2.2.1'
+version          '2.2.2'
```

Comment:
We would prefer to do the version bumping after a merge, could you please revert this change?